contracts/Homework.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.3;
```

Comment:
It's a good practice to hardcode compiler version for security reasons. The exception is when you're building a library contract that is to be imported by many other contracts. In such cases you aim at the widest range of versions possible.
```suggestion
pragma solidity 0.8.3;
```